logcheck doesn't handle Exit* functions correctly

/kind bug

klog defines set of functions Exit, Exitf, Exitln, ExitDepth. Those functions are not correctly handled by logcheck. https://github.com/kubernetes/kubernetes/pull/103293#issuecomment-885028839

**What steps did you take and what happened:**

When running logcheck to validate log calls, it reported Exitf as structured logging method
```
/home/prow/go/src/k8s.io/kubernetes/test/images/apparmor-loader/loader.go:64:3: structured logging function "Exitf" should not use format specifier "%s"
```

**What did you expect to happen:**

I would expect that logcheck will ignore this method when in `allow-unstructured` mode and report it as error only when `allow-unstructured` is off.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

--- a/hack/tools/logcheck/main.go
+++ b/hack/tools/logcheck/main.go
@@ -129,6 +129,7 @@ func isUnstructured(fName string) bool {
 		"Warning", "Warningf", "Warningln", "WarningDepth",
 		"Error", "Errorf", "Errorln", "ErrorDepth",
 		"Fatal", "Fatalf", "Fatalln", "FatalDepth",
+		"Exit", "Exitf", "Exitln", "ExitDepth",
 	}
 
 	for _, name := range unstrucured {

--- a/hack/tools/logcheck/testdata/src/allowUnstructuredLogs/allowUnstructuredLogs.go
+++ b/hack/tools/logcheck/testdata/src/allowUnstructuredLogs/allowUnstructuredLogs.go
@@ -60,4 +60,8 @@ func allowUnstructuredLogs() {
 	klog.Fatalf("test log")
 	klog.Fatalln("test log")
 	klog.FatalDepth(1, "test log")
+	klog.Exit("test log")
+	klog.ExitDepth(1, "test log")
+	klog.Exitln("test log")
+	klog.Exitf("test log")
 }

--- a/hack/tools/logcheck/testdata/src/doNotAllowUnstructuredLogs/doNotAllowUnstructuredLogs.go
+++ b/hack/tools/logcheck/testdata/src/doNotAllowUnstructuredLogs/doNotAllowUnstructuredLogs.go
@@ -60,5 +60,8 @@ func doNotAllowUnstructuredLogs() {
 	klog.Fatalf("test log")          // want `unstructured logging function "Fatalf" should not be used`
 	klog.Fatalln("test log")         // want `unstructured logging function "Fatalln" should not be used`
 	klog.FatalDepth(1, "test log")   // want `unstructured logging function "FatalDepth" should not be used`
-
+	klog.Exit("test log")            // want `unstructured logging function "Exit" should not be used`
+	klog.Exitf("test log")           // want `unstructured logging function "Exitf" should not be used`
+	klog.Exitln("test log")          // want `unstructured logging function "Exitln" should not be used`
+	klog.ExitDepth(1, "test log")    // want `unstructured logging function "ExitDepth" should not be used`
 }

--- a/hack/tools/logcheck/testdata/src/k8s.io/klog/v2/klog.go
+++ b/hack/tools/logcheck/testdata/src/k8s.io/klog/v2/klog.go
@@ -180,3 +180,20 @@ func Fatalln(args ...interface{}) {
 // Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
 func Fatalf(format string, args ...interface{}) {
 }
+
+func Exit(args ...interface{}) {
+}
+
+// ExitDepth acts as Exit but uses depth to determine which call frame to log.
+// ExitDepth(0, "msg") is the same as Exit("msg").
+func ExitDepth(depth int, args ...interface{}) {
+}
+
+// Exitln logs to the FATAL, ERROR, WARNING, and INFO logs, then calls os.Exit(1).
+func Exitln(args ...interface{}) {
+}
+
+// Exitf logs to the FATAL, ERROR, WARNING, and INFO logs, then calls os.Exit(1).
+// Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.
+func Exitf(format string, args ...interface{}) {
+}

